### ISSUE TITLE
Feature Request:  addin to assist w/config build

### ISSUE THREAD

**Issue by nicholas-masel** [pharmaverse/envsetup#13]
### Feature Idea

Create a utility to build a configuration file by allowing users to:

1.  name a path, browse through their folder structure and select the path or paths that should be searched.
2.  select paths where functions are automatically sourced.
3. define where to store the configuration file.
4. "activate" the configuration by adding the relevant envsetup code to the .Rprofile (in a similar fashion to renv).

**Comment by nicholas-masel** [comment]
I don't see a big need for this unless we want to try to have regulatory use this to re-setup an environment. If we have a use case, we can add a new issue and add this feature. @mstackhouse are you ok with this and I'll close it?

**Comment by mstackhouse** [comment]
Yeah let's close - fun idea but not the first place I'd focus effort.
